### PR TITLE
Changelog for v2.7.2

### PR DESCRIPTION
**Pull request by 6543**

### CODE REVIEW

**Comment by 6543** [comment]
@qwerty287 yes but it just adds a new debug line to acktually tackle down (config-) bugs (actual users had in prod) ...

... so we should release it for everyone but I'm not sure if it is even worth the changelog ...

If this is blocking I just make it not listed in the changelog ...

**Comment by 6543** [comment]
@qwerty287 did that address your concerns?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+
+@6543, @anbraten, @j04n-f, @qwerty287
+
+### 🔒 Security
+
+- Chore(deps): update dependency vite to v5.4.6 [security] ([#4163](https://github.com/woodpecker-ci/woodpecker/pull/4163)) [[#4187](https://github.com/woodpecker-ci/woodpecker/pull/4187)]
+
+### 🐛 Bug Fixes
+
+- Run queue.process() in background [[#4115](https://github.com/woodpecker-ci/woodpecker/pull/4115)]
```
Comment (6543) [inline CHANGELOG.md:21]:
```suggestion
- Fix repo/owner parsing for gitlab ([#4255](https://github.com/woodpecker-ci/woodpecker/pull/4255)) [[#4261](https://github.com/woodpecker-ci/woodpecker/pull/4261)]
- Run queue.process() in background [[#4115](https://github.com/woodpecker-ci/woodpecker/pull/4115)]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+
+### Important
+
+If you want a secure instance, set the `WOODPECKER_PLUGINS_PRIVILEGED` to only allow `woodpeckerci/plugin-docker-buildx` plugin to be v5.0.0 or later.
+
+[Read more](https://github.com/woodpecker-ci/woodpecker/pull/4213)
+
+### ❤️ Thanks to all contributors! ❤️
+
+@6543, @anbraten, @j04n-f, @qwerty287
```
Comment (6543) [inline CHANGELOG.md:13]:
```suggestion
@6543, @anbraten, @j04n-f, @pat-s, @qwerty287
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## [2.7.2](https://github.com/woodpecker-ci/woodpecker/releases/tag/v2.7.1) - 2024-10-09
```
Comment (anbraten) [inline CHANGELOG.md:3]:
```suggestion
## [2.7.2](https://github.com/woodpecker-ci/woodpecker/releases/tag/v2.7.2) - 2024-10-09
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+
+### Important
+
+If you want a secure instance, set the `WOODPECKER_PLUGINS_PRIVILEGED` to only allow `woodpeckerci/plugin-docker-buildx` plugin to be v5.0.0 or later.
```
Comment (anbraten) [inline CHANGELOG.md:7]:
````suggestion
To secure your instance, set `WOODPECKER_PLUGINS_PRIVILEGED` to only allow specific versions of the `woodpeckerci/plugin-docker-buildx` plugin, use version 5.0.0 or above. This prevents older, potentially unstable versions from being privileged.

For example, to allow only version 5.0.0, use:

```bash
WOODPECKER_PLUGINS_PRIVILEGED=woodpeckerci/plugin-docker-buildx:5.0.0
```

To allow multiple versions, you can separate them with commas:

```bash
WOODPECKER_PLUGINS_PRIVILEGED=woodpeckerci/plugin-docker-buildx:5.0.0,woodpeckerci/plugin-docker-buildx:5.1.0
``` 

This setup ensures only specified, stable plugin versions are given privileged access.
````

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,31 @@
...
+
+[Read more](https://github.com/woodpecker-ci/woodpecker/pull/4213)
```
Comment (anbraten) [inline CHANGELOG.md:9]:
```suggestion
Read more about it in [#4213](https://github.com/woodpecker-ci/woodpecker/pull/4213)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## [2.7.2](https://github.com/woodpecker-ci/woodpecker/releases/tag/v2.7.2) - 2024-10-09
```
Comment (6543) [inline CHANGELOG.md:3]:
```suggestion
## [2.7.2](https://github.com/woodpecker-ci/woodpecker/releases/tag/v2.7.2) - 2024-10-28
```

**Comment by anbraten** [comment]
> No. If you think this really should go into a patch release, I won't block it, but please add it to the changelog.

We can just release this as `2.8.0`?

**Comment by anbraten** [comment]
I would also like to include https://github.com/woodpecker-ci/woodpecker/pull/3348. As explained [here](https://github.com/woodpecker-ci/woodpecker/discussions/4234#discussioncomment-10990275), our current approach to limiting secret access and protecting through plugin image filters is effective but somewhat prone to occasional issues. Introducing a system that requires approvals for executing potentially risky pipelines - such as forked pull requests - beforehand should greatly reduce these risks.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,45 @@
...
+
+@6543, @anbraten, @j04n-f, @pat-s, @qwerty287
+
+### 🔒 Security
+
+- Chore(deps): update dependency vite to v5.4.6 [security] ([#4163](https://github.com/woodpecker-ci/woodpecker/pull/4163)) [[#4187](https://github.com/woodpecker-ci/woodpecker/pull/4187)]
+
+### 🐛 Bug Fixes
+
+- Fix repo/owner parsing for gitlab ([#4255](https://github.com/woodpecker-ci/woodpecker/pull/4255)) [[#4261](https://github.com/woodpecker-ci/woodpecker/pull/4261)]
```
Comment (6543) [inline CHANGELOG.md:35]:
```suggestion
- Fix repo/owner parsing for gitlab ([#4272](https://github.com/woodpecker-ci/woodpecker/pull/4272)) [[#4273](https://github.com/woodpecker-ci/woodpecker/pull/4273)]
- Fix repo/owner parsing for gitlab ([#4255](https://github.com/woodpecker-ci/woodpecker/pull/4255)) [[#4261](https://github.com/woodpecker-ci/woodpecker/pull/4261)]
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,46 @@
 # Changelog
 
+## [2.7.2](https://github.com/woodpecker-ci/woodpecker/releases/tag/v2.7.2) - 2024-10-28
```
Comment (6543) [inline CHANGELOG.md:3]:
```suggestion
## [2.7.2](https://github.com/woodpecker-ci/woodpecker/releases/tag/v2.7.2) - 2024-10-29
```

**Comment by 6543** [comment]
Well i also want the batching of loglines also released asap ... so i think we can do v2.8.0 soon?

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,46 @@
 # Changelog
 
+## [2.7.2](https://github.com/woodpecker-ci/woodpecker/releases/tag/v2.7.2) - 2024-10-29
```
Comment (6543) [inline CHANGELOG.md:3]:
```suggestion
## [2.7.2](https://github.com/woodpecker-ci/woodpecker/releases/tag/v2.7.2) - 2024-11-02
```

**Comment by anbraten** [comment]
@6543  13:00? => https://github.com/woodpecker-ci/woodpecker/pull/4291

**Pull request merged**